app/helpers/application_helper/toolbar/middleware_datasources_center.rb

```diff
@@ -37,7 +37,9 @@ class ApplicationHelper::Toolbar::MiddlewareDatasourcesCenter < ApplicationHelpe
             :url_parms => "main_div",
             :enabled   => false,
             :onwhen    => "1+",
-            :confirm   => N_('Do you want to remove these datasources ?'))
+            :confirm   => N_('Do you want to remove these Datasources ? Some Applications could be using these '\
```

Comment:
everywhere else seems to be like that, so I left it that way.
https://github.com/josejulio/manageiq/blob/2c6870ed583e79680209d5b005790021f84d4696/app/helpers/application_helper/toolbar/middleware_deployments_center.rb#L40

Do you think I should use this PR to fix that? (I don't mind fixing it)